x-pack/plugin/rollup/src/test/java/org/elasticsearch/xpack/rollup/v2/RollupActionSingleNodeTests.java

```diff
@@ -112,13 +112,12 @@ protected Collection<Class<? extends Plugin>> getPlugins() {
 
     @Before
     public void setup() {
-        sourceIndex = randomAlphaOfLength(5).toLowerCase(Locale.ROOT);
-        sourceIndexClone = sourceIndex + "-clone";
-        rollupIndex = randomAlphaOfLength(6).toLowerCase(Locale.ROOT);
+        sourceIndex = randomAlphaOfLength(12).toLowerCase(Locale.ROOT);
+        rollupIndex = "rollup-" + sourceIndex + "-" + randomAlphaOfLength(4).toLowerCase(Locale.ROOT);
```

Comment:
Making the source index 12 chars long make it extremely impossible to hit a conflict.
Also, changed the rollup index name so that it cannot conflict with source index name. 
Bonus, it's easy to tell which index is the source and which is is the rollup index at the logs

Reply:
What about `getTestName()`?